src/custom/components/Tokens/styled.ts

```diff
@@ -4,11 +4,19 @@ import { ThemedText } from 'theme'
 import { LightCard } from 'components/Card'
 import { AutoColumn } from 'components/Column'
 import CurrencyLogo from 'components/CurrencyLogo'
+import { SearchInput } from 'components/SearchModal/styleds'
+
+export const TokenSearchInput = styled(SearchInput)`
```

Comment:
On the dark theme, I can't really see the search input border since its too dark
![Screenshot from 2022-06-08 13-36-26](https://user-images.githubusercontent.com/34926005/172613874-b973deec-9411-423e-a70e-9715d5418196.png)

Reply:
fixed